cityofaustin/atd-moped
# Project Components: Extend "Map Project" step to include component attributes #6250

Closes https://github.com/cityofaustin/atd-data-tech/issues/6250

** Test in moped-test environment (this PR includes db changes and front-end changes)
As of today, it's been deployed here: https://moped-test.austinmobility.io/moped/projects/227?tab=map

To Test:
1. Deploy in moped-test (or locally) 
2. Visit: https://moped-test.austinmobility.io/moped
3. Create a new project
4. The map should automatically create a new component
5. Make changes to the component
6. The changes should reflect in the maps

Moped-Test:

https://user-images.githubusercontent.com/5282430/123328725-8b149e80-d501-11eb-9250-121736e3f144.mp4
